lib/upipe-blackmagic/upipe_blackmagic_source.cpp

```diff
@@ -821,8 +821,10 @@ static int upipe_bmd_src_set_uri(struct upipe *upipe, const char *uri)
     }
 
     const char *model_name;
-    if (deckLink->GetModelName(&model_name) == S_OK)
+    if (deckLink->GetModelName(&model_name) == S_OK) {
         upipe_notice_va(upipe, "detected card type %s", model_name);
+        free((void*)model_name);
```

Comment:
Then it's not a const char * ? The SDK doc says string *, any idea what this represents ? Otherwise change const char * to char * to avoid the cast. Perhaps we should use delete instead of free as well.